src/model/User.java

```diff
@@ -0,0 +1,5 @@
+package model;
+
+abstract public class User extends ID {
```

Comment:
User is not an ID (OOP inheritance principle). Rename the class ID or move this code to some interface, for example "Identifiable"

Reply:
done